### PR TITLE
Edit readme

### PR DESCRIPTION
**Pull request by kinsaurralde**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,49 @@
-This is the Videomap capstone project
+# Videomap
+
+<img src="https://user-images.githubusercontent.com/36008213/91629325-ee36b280-e97c-11ea-9ec8-994a088b64a7.png" width=500>
+<img src="https://user-images.githubusercontent.com/55601789/91622207-dfcfa300-e94a-11ea-83a6-dd8b2095e58e.gif" width=500>
+<img src="https://user-images.githubusercontent.com/36008213/90663960-33314b00-e1ff-11ea-9790-fa5883098d59.png" width=500>
+
+## About
+Videomap is an interactive map that comnbines Google Maps and Youtube to provide a simple way for users to find videos related to location. Videomap includes additional features related to COVID-19 including:
```
Comment (panerorenn9541) [inline README.md]: *combines on line 8

File: README.md [hunk 2]
```diff
@@ -1 +1,49 @@
-This is the Videomap capstone project
+# Videomap
```
Comment (panerorenn9541) [inline README.md:1]: Maybe include the deployed website link somewhere (http://videomap-step-2020.appspot.com/)

**Review by panerorenn9541 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (alsile) [reply README.md:1]: second this, could be in parenthesis by # Videomap (i.e # Videomap (link))

**Review by alsile — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1 +1,50 @@
...
+[http://videomap-step-2020.appspot.com/](http://videomap-step-2020.appspot.com/)
+
+<img src="https://user-images.githubusercontent.com/36008213/91629325-ee36b280-e97c-11ea-9ec8-994a088b64a7.png" width=500>
+<img src="https://user-images.githubusercontent.com/55601789/91622207-dfcfa300-e94a-11ea-83a6-dd8b2095e58e.gif" width=500>
+<img src="https://user-images.githubusercontent.com/36008213/90663960-33314b00-e1ff-11ea-9790-fa5883098d59.png" width=500>
```
Comment (huzcn) [inline README.md]: I really like the second gif which acts like a short demo for our project. Can you (or others) replace the other two with gif as well?

File: README.md [hunk 4]
```diff
@@ -1 +1,50 @@
...
+
+## About
+Videomap is an interactive map that combines Google Maps and Youtube to provide a simple way for users to find videos related to location. Videomap includes additional features related to COVID-19 including:
+- statistical and graphical data related to the virus
```
Comment (huzcn) [inline README.md:10]: related to the corona coronavirus pandemic

File: README.md [hunk 5]
```diff
@@ -1 +1,50 @@
...
+- routing with COVID-19 information
+- heat map functionality based on various COVID-19 data points (confirmed, active, deaths, recovered, per capita)
```
Comment (huzcn) [inline README.md]: I prefer more depths to each of these features. Feel free to copy from the design doc if there is any.
You can have Noe and Eyad to help elaborate these in follow-on pull requests.

**Review by huzcn — CHANGES_REQUESTED** [review]
Nice! It's better to also add the framework diagram.

**Review by huzcn — APPROVED** [review]
Please continue adding more details to README.
@alsile @panerorenn9541

**Pull request merged**